Pocket Casts Watch App Extension/PodcastEpisodeListView.swift

```diff
@@ -8,11 +8,14 @@ struct PodcastEpisodeListView: View {
     @StateObject var viewModel: PodcastEpisodeListViewModel
 
     var body: some View {
-        ScrollView {
-            LazyVStack {
-                podcastInfo
-                EpisodeListView(title: L10n.podcastsPlural.prefixSourceUnicode, showArtwork: false, episodes: $viewModel.episodes, playlist: .podcast(uuid: viewModel.podcast.uuid))
-            }
+        List {
+            Section(
+                content: {
+                    EpisodeListView(title: L10n.podcastsPlural.prefixSourceUnicode, showArtwork: false, episodes: $viewModel.episodes, playlist: .podcast(uuid: viewModel.podcast.uuid))
+                },
+                header: {
+                    podcastInfo.textCase(.none)
+                })
```

Comment:
[nit] We can omit the `content:` here 

```suggestion
            Section {
                EpisodeListView(title: L10n.podcastsPlural.prefixSourceUnicode, showArtwork: false, episodes: $viewModel.episodes, playlist: .podcast(uuid: viewModel.podcast.uuid))
            } header: {
                podcastInfo.textCase(.none)
            }
```